### PR TITLE
3rd commit Homework4 2ndBranch

### PR DESCRIPTION
**Pull request by Jalmer13**

### CODE REVIEW

File: firstapp/Homework4.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
+package firstapp;
+
+public class Homework4 {
+    public static void main(String[] args) {
+        System.out.println("Task 1 CheckOddEven");
+        int fstnum = 12;
```
Comment (AndrewSkoba) [inline firstapp/Homework4.java]: vars name should be in camel case format.
For example firstNumber

Also, check below

File: firstapp/Homework4.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        int sndnum = 4;
+        double result = fstnum % sndnum;
+        if (result != 0) {
+            System.out.println("Odd number");
+        } else if (result == 0) {
```
Comment (AndrewSkoba) [inline firstapp/Homework4.java]: you can just use else statement instead

File: firstapp/Homework4.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            System.out.println("AUG");
+        } else if (month == 9) {
+            System.out.println("SEPT");
+        } else if (month == 10) {
+            System.out.println("OCT");
+        } else if (month == 11) {
+            System.out.println("NOV");
+        } else if (month == 12) {
+            System.out.println("DEC");
+        } else if (month > 12 && month < 1 ) {
```
Comment (AndrewSkoba) [inline firstapp/Homework4.java]: Also, just else statement

File: firstapp/Homework4.java [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        System.out.println("Task 3 Absolute value");
+        int trdnum = 3;
+        int fthnum = -9;
+        int abs1 = Math.abs(trdnum);
+        int abs2 = Math.abs(fthnum);
+        if (abs1 > abs2) {
+            System.out.println(trdnum + " has a greater absolute value than " + fthnum);
+        } else if (abs1 < abs2) {
+            System.out.println(fthnum + " has a greater absolute value than " + trdnum);
+        } else if (abs1 == abs2) {
```
Comment (AndrewSkoba) [inline firstapp/Homework4.java]: just else

File: firstapp/Homework4.java [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            System.out.println(trdnum + " has equal absolute value as " + fthnum);
+        }
+
+        System.out.println("Task 4 Isosceles triangle");
+        int sideA = 5;
+        int sideB = 5;
```
Comment (AndrewSkoba) [inline firstapp/Homework4.java:106]: Triangle has 3 sides, and you do not know exactly what the sides are equal.
Lets imagine that sideA != sideB but sideA == sideC - as a result triangle isosceles

File: firstapp/Homework4.java [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+        } else if (sideA != sideB) {
+            System.out.println("Triangle is not isosceles");
+        }
+
+        System.out.println("Task 5 Middle symbols");
+        String str1 = "Hillel";
+        String str2 = "Java";
+        String str3 = "Hometask";
+
+        System.out.println("Middle symbols of " + str1 + " is " + str1.substring(2,4));
```
Comment (AndrewSkoba) [inline firstapp/Homework4.java]: This solution is hardcoded a bit
Would be better to write some logic to find indexes(begin and end) for substring method

File: firstapp/Homework4.java (hunk 2 above)
Comment (Jalmer13) [reply firstapp/Homework4.java]: idea пишет, что else без if не statement
Screenshot_4
*понял замечание

**Review by AndrewSkoba — APPROVED** [review]

**Pull request closed without merging**